ZenPacks/zenoss/WBEM/modeler/wbem.py

```diff
@@ -186,3 +207,25 @@ def handle_failure(failure):
     deferred.addErrback(handle_failure)
 
     return deferred
+
+
+def check_if_complete(results, log, device, namespace, classname, _results=None):
```

Comment:
You have to shed more light on what the difference between the results and _results attributes in a docstring or use appropriate names.

Reply:
Also, the line is longer than 79 characters.

Reply:
I would expect more comments in the function as it either returns a deferred or some results.